src/shared/ui/dropdown/dropdown-provider.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react'
+import DropDownTrigger from './dropdown-trigger'
+import DropDownMenu from './dropdown-menu'
+import DropDownMenuItem from './dropdown-menu-ltem'
+import { DropdownProvider } from './hooks/useDropDownContext'
+/**
+ * @description 드롭다운 UI 컴포넌트
+ * DropdownProvider로 context를 하위 컴포넌트에 전달합니다
+ *
+ * <하위컴포넌트>
+ * DropDown.Trigger - 드롭다운 트리거 버튼
+ * DropDown.Menu - 드롭다운 메뉴 상자
+ * Dropdown.Item - 드롭다운 메뉴 리스트
+ *
+ * @example
+ *    <Dropdown>
+ *        <Dropdown.Trigger>드롭다운 트리거 버튼</Dropdown.Trigger>
+ *        <Dropdown.Menu width="sm">
+ *          <Dropdown.Item>목록1</Dropdown.Item>
+ *          <Dropdown.Item>목록2</Dropdown.Item>
+ *        </Dropdown.Menu>
+ *    </Dropdown>
+ *
+ */
+
+export default function Dropdown({ children }: { children: React.ReactNode }) {
+  const [isOpen, setIsOpen] = useState(false)
+  const handleDropdown = () => setIsOpen((prev) => !prev)
+  const closeDropdown = () => setIsOpen(false)

```

Comment:
p2

핸들러 함수도 사용하게 되면 이름을 통일하는게 좋을 것 같아요

위에는 단순히 `handleDropdown` 이고 아래는 `closeDropdown`인데 
핸들러 함수라면 동일하게 handle을붙이고 무슨 동작을 할건지도 적어주면 어떨까요?

```suggestion
  const handleToggleDropdown = () => setIsOpen((prev) => !prev)
  const handleCloseDropdown = () => setIsOpen(false)
```